js-miniapp-bridge/src/types/custom-permissions.ts

```diff
@@ -4,6 +4,7 @@ export enum CustomPermissionName {
   CONTACT_LIST = 'rakuten.miniapp.user.CONTACT_LIST',
   ACCESS_TOKEN = 'rakuten.miniapp.user.ACCESS_TOKEN',
   LOCATION = 'rakuten.miniapp.device.LOCATION',
+  SEND_MESSAGE = 'rakuten.miniapp.action.SEND_MESSAGE',
```

Comment:
@corycaywood Are we planning to use the same name in Android & iOS ?

As of now, in iOS it is `rakuten.miniapp.user.SEND_MESSAGE`, I was suggesting that we will change it to `rakuten.miniapp.user.action.SEND_MESSAGE`.

What do you think ?

Reply:
Yeah, it should be the same on both platforms. I don't really have an opinion on which one we should use. @minh-rakuten What do you think?

Reply:
We will use `rakuten.miniapp.user.action.SEND_MESSAGE`. cc @khairul-alam-licon

Reply:
Ok, I'll update Android using `rakuten.miniapp.user.action.SEND_MESSAGE`